code/API_definitions/qos-profiles.yaml

```diff
@@ -510,12 +487,21 @@ components:
       maxProperties: 4
 
     NetworkAccessIdentifier:
-      description: A public identifier addressing a subscription in a mobile network. In 3GPP terminology, it corresponds to the GPSI formatted with the External Identifier ({Local Identifier}@{Domain Identifier}). Unlike the telephone number, the network access identifier is not subjected to portability ruling in force, and is individually managed by each operator.
+      description: |
+        A public identifier for a mobile network subscription. In simple
+        terms, it follows the format "Local Identifier@Domain Identifier".
+        Unlike a telephone number, this identifier is fixed to one operator
+        and cannot be transferred between providers.
       type: string
       example: "123456789@domain.com"
 
     PhoneNumber:
-      description: A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, prefixed with '+'.
+      description: |
+        This field represents a telephone number that uniquely identifies a mobile subscription.
+        It corresponds to the MSISDN (Mobile Station International Subscriber Directory Number)
+        used in mobile networks. The number must be provided in international format, starting
+        with a '+' followed by the country code and the subscriber number, in accordance with
+        the E.164 standard.
```

Comment:
Same as above ... we should first introduce a change to Commonalities

Reply:
reverted this change